spec/index.bs

```diff
@@ -13129,8 +13128,14 @@ be rare and can safely be ignored. Applications should not be written in such a
 timestamps cause an application failure.
 
 <p tracking-vector>
-Timestamp queries can provide high-resolution GPU timing.
-See [[#security-timing]] for security considerations.
+Timestamp queries can provide high-resolution GPU timing. To mitigate privacy concerns, they must be
+processed with the following algorithm:
+
+<div algorithm>
+    <dfn abstract-op>Process timestamp value</dfn>
```

Comment:
We need to reference this from the pseudocode which implements `writeTimestamp` and `GPU{Render,Compute}TimestampWrites.{beginning,end}OfPassWriteIndex`.

I think we should write this algorithm as `To get the <dfn abstract-op>current queue timestamp</dfn>:` and then replace the places where it currently says "current timestamp for the Queue timeline" or "current queue timestamp".

Reply:
Done